ark/phenotyping/som_utils.py

```diff
@@ -271,9 +273,13 @@ def compute_pixel_cluster_channel_avg(fovs, channels, base_dir, pixel_cluster_co
 
     for fov in fovs:
         # read in the fovs data
-        fov_pixel_data = feather.read_dataframe(
-            os.path.join(base_dir, pixel_data_dir, fov + '.feather')
-        )
+        try:
+            fov_pixel_data = feather.read_dataframe(
+                os.path.join(base_dir, pixel_data_dir, fov + '.feather')
+            )
+        except (ArrowInvalid, OSError, IOError):
+            print("The data for FOV %s has been corrupted, removing" % fov)
```

Comment:
I think "skipping" is more accurate, since we're not actually removing it. 

The user will need to re-run the whole pipeline for this specific FOV. We should give them an error message indicating that, and also delete this specific feather file so that when they re-generate it there's no overwriting issues